### PR TITLE
Remaining destinations revamp for modals topics

### PR DESCRIPTION
**Pull request by jamescott**
Update formatting for Active Campaign modal + external links formatting + add Adobe Marketo credentials.

### CODE REVIEW

File: amperity_datagrid/source/destination_active_campaign.rst [hunk 1]
```diff
@@ -115,7 +115,7 @@ Get details
           :alt: Detail four.
           :align: left
           :class: no-scaled-link
-     - The name of a |ext_active_campaign_contacts_list| in |destination-name|. This can be the name of an existing list or a new list.
+     - The name of a `contacts list <https://help.activecampaign.com/hc/en-us/articles/360000030559-How-to-create-a-list-in-ActiveCampaign>`__ |ext_link| in |destination-name|. This can be the name of an existing list or a new list.
```
Comment (karenkathryn) [inline amperity_datagrid/source/destination_active_campaign.rst:118]: Nit - Consider "list of contacts"

**Review by Jacksonbaker323 — APPROVED** [review]
Yep

File: amperity_datagrid/source/destination_active_campaign.rst [hunk 2]
```diff
@@ -345,7 +345,7 @@ Add data template
           * - **Setting**
             - **Description**
           * - **List name**
-            - The name of a |ext_active_campaign_contacts_list| in |destination-name|. This can be the name of an existing list or a new list.
+            - The name of a contacts list in |destination-name|. This can be the name of an existing list or a new list.
```
Comment (karenkathryn) [inline amperity_datagrid/source/destination_active_campaign.rst:348]: Nit: list of contacts

File: amperity_datagrid/source/source_salesforce_sales_cloud.rst [hunk 3]
```diff
@@ -67,8 +67,7 @@ Get details
 
 Salesforce Sales Cloud requires the following configuration details:
 
-#. The Salesforce Sales Cloud username.
-#. The Salesforce Sales Cloud password.
+#. The username and password of a Salesforce account configured for API access.
 #. The Salesforce Sales Cloud security token that belongs to username. (The security token is not required if IP range policies are configured from the Salesforce admin console.)
 #. The scheme and host for a custom Salesforce Sales Cloud URL, if used by customer.
```
Comment (karenkathryn) [inline amperity_datagrid/source/source_salesforce_sales_cloud.rst:72]: "If used by *a* customer"

**Review by karenkathryn — APPROVED** [review]
Minor comments from brief review

**Pull request merged**